ViewModel/TableWithPlot/ComputeComponentVM.cs

```diff
@@ -55,6 +55,7 @@ public string XLabel
             {
                 _xlabel = value;
                 NotifyPropertyChanged();
+                NotifyPropertyChanged(nameof(Units));
```

Comment:
Why add another notify prop changed here?

Reply:
The label on the UI is bound to the property "Units" which takes XLabel and YLabel and formats them into a nice string to display. I update Y label when I switch between stage and discharge, so i need units to change when y label changes